### DISCUSSION TITLE
Acknowledging snap maintainence

### DISCUSSION THREAD

**Discussion by ghost** [tsujan/FeatherPad#745] [General]
Hey @tsujan  i wish to publish featherpad as a snap as a downstream maintainer, but the snap name is withheld by someone who is inactive for a very long time and am and the store review team is unable to contact him, in snap community, upstream snap names are reserved for upstream and is only allowed if the upstream agrees to let  someone build the snap package, so can you please acknowledge here letting me build and publish featherpad as a 3rd party package(snaps) , i will undertake all the responsibilities for the package regarding issues etc.
See--> https://launchpad.net/lxqt-snaps (the proposed issue/bug reporter site)

P.S. I will clearly specify it as 3rd party package with no direct relations with the upstream and will undertake all responsibilities of its bug management etc.

**Comment by tsujan** [comment]
> i will undertake all the responsibilities for the package regarding issues etc.

Thanks!

The latest version is 1.4.1, released a few days ago.

**Reply by tsujan** [reply]
I appreciate your responsible approach.

> ... upstream snap names are reserved for upstream and is only allowed if the upstream agrees to let someone build the snap package

This doesn't seem logical. As the developer of FeatherPad, I don't participate in *any kind of packaging*. The reason is clear: I'm not responsible for them.

Since you're packaging FeatherPad in a certain way, you don't need my permission — you have my thanks instead.

**Reply by ghost** [reply]
Welcome, i know but this is how snap community works.

**Reply by tsujan** [reply]
> ... but this is how snap community works.

Then they need to change it. It isn't good to pretend that an upstream developer has any kind of responsibility for a package.

**Reply by ghost** [reply]
I am telling them the same regarding other lxqt packages too, hope this will get changed.

**Reply by tsujan** [reply]
Just for the clarification: The LXQt team isn't responsible either for snap LXQt packages, although the more packages, the better.

BTW, FeatherPad isn't a part of LXQt. Yes, I'm an LXQt developer, but I develop my own apps independently of LXQt.

**Reply by ghost** [reply]
I know, thanks for the clarification, i believe i will have to create this similar request in the main lxqt git repo so that that they will let me snap it, and as such i also believe in taking responsibility for the work one do.

**Reply by tsujan** [reply]
> i believe i will have to create this similar request in the main lxqt git repo

Please do it in https://github.com/lxqt/lxqt/discussions

**Reply by ghost** [reply]
Ok

**Comment by stefonarch** [comment]
To avoid confusion: featherpad shouldn't be included under "LXQt snaps", as mentioned by @tsujan it's not a LXQt apps like Qps or screengrab or LXimage-qt.

**Reply by ghost** [reply]
Ok i will create a new project hosting exclusively feather* apps, thanks.

**Comment by ghost** [comment]
Done, feather*  apps have been shifted to -->  https://launchpad.net/miscellaneous-snaps

**Comment by brlin-tw** [comment]
Hello, first of all, apologies for the ignorance.  As the discussion is currently scattered throughout multiple places([downstream snap project issue report](https://github.com/brlin-tw/featherpad-snap/issues/11), [upstream project issue report](https://github.com/tsujan/FeatherPad/discussions/745), [Snapcraft Forum snap ownership transfer post](https://forum.snapcraft.io/t/transfer-featherpad/35635/2), and the Snap Store private backstage messages),  I would like to consolidate them into [this Snapcraft Forum thread ](https://forum.snapcraft.io/t/transfer-featherpad/35635/3)as it is the proper place for the matter to be discussed in the first place.  I'll also forward the necessary portions of responses to the upstream issue report with full-quoted context as not everyone has a Snapcraft Forum account.

Thank you for your understanding!
